bin/varnishd/cache/cache_backend.c

```diff
@@ -325,9 +325,15 @@ vbe_dir_getfd(VRT_CTX, struct worker *wrk, VCL_BACKEND dir, struct backend *bp,
 
 	PFD_LocalName(pfd, abuf1, sizeof abuf1, pbuf1, sizeof pbuf1);
 	PFD_RemoteName(pfd, abuf2, sizeof abuf2, pbuf2, sizeof pbuf2);
-	VSLb(bo->vsl, SLT_BackendOpen, "%d %s %s %s %s %s %s",
-	    *fdp, VRT_BACKEND_string(dir), abuf2, pbuf2, abuf1, pbuf1,
-	    PFD_State(pfd) == PFD_STATE_STOLEN ? "reuse" : "connect");
+	if (PFD_State(pfd) != PFD_STATE_STOLEN) {
+		VSLb(bo->vsl, SLT_BackendOpen, "%d %s %s %s %s %s connect",
+		    *fdp, VRT_BACKEND_string(dir), abuf2, pbuf2, abuf1, pbuf1);
+	} else {
+		VSLb(bo->vsl, SLT_BackendOpen,
+		    "%d %s %s %s %s %s reuse %.6f %ju", *fdp,
+		    VRT_BACKEND_string(dir), abuf2, pbuf2, abuf1, pbuf1,
+		    PFD_Age(pfd, VTIM_real()), PFD_Reused(pfd));
```

Comment:
There's no point having a `now` argument in `PFD_Age()` if we end up passing `VTIM_real()`, we just create ceremony. My expectation was that we should have a `ctx->now`, or one `ctx->wrk->lastused` freshly computed from the last timestamp update.

If we are not reusing a real timestamp available in this scope then there is no point having a `now` argument or using real time at all.

Reply:
I have for now dropped the `now` argument. In the future we might add it back as `vtim_mono`, but let's not delve into this right now.